plugins/woocommerce/src/Internal/EmailEditor/templates/WooMailTemplate.php

```diff
@@ -0,0 +1,36 @@
+<?php declare(strict_types = 1);
+
+namespace Automattic\WooCommerce\Internal\EmailEditor\Templates;
+
+class WooMailTemplate {
+
+	public function get_slug(): string {
+		return 'woomailtemplate';
+	}
+
+	public function get_title(): string {
+		return __('WooMailTemplate', 'woocommerce');
```

Comment:
Shall we rather use the `Woo Email Template"?
I think it would look nicer in UIs like this one:
![Screenshot 2025-02-24 at 13 22 26](https://github.com/user-attachments/assets/0d192b4e-7c89-45ab-8fa4-e2c4fde8649d)

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/55754/commits/c938f12469ee29c0e6b92f752f34600c87ea7648